### ISSUE TITLE
Argument type MimeType

### ISSUE THREAD

**Issue by yasserlassance** [gualtierofrigerio/WKDownloadHelper#4]
SupportedMimeTypes cannot convert value of type MimeType to  mimeTypes
`let mimeTypes = [MimeType(type: "ms-excel", fileExtension: "xls"),
                         MimeType(type: "pdf", fileExtension: "pdf")]
        downloadHelper = WKDownloadHelper(webView: webView,
                                          supportedMimeTypes: mimeTypes,
                                          delegate: self)
    let request = URLRequest(url: URL(string: "https://www.google.it")!)
    webView.load(request)`

**Comment by gualtierofrigerio** [comment]
It this a compilation error? I don't see it in my sample project if I copy paste those lines with tag 0.3.0

**Comment by yasserlassance** [comment]
> It this a compilation error? I don't see it in my sample project if I copy paste those lines with tag 0.3.0

Yes, it's a compilation error, in ViewController.swift. I added your package in my project and copied your viewDidLoad() to test but it shows this error.

1. Arguments to generic parameter 'Element' ('Project.MimeType' and 'WKDownloadHelper.MimeType') are expected to be equal.

**Comment by gualtierofrigerio** [comment]
Looks like a name conflict. Do  you have another MimeType struct or class in your project?
If you do, I can suggest a workaround. You can try importing only MimeType from WKDownloadHelper and make a type alias, like this
```swift
import struct WKDownloadHelper.MimeType
typealias WKDownloadMimeType = WKDownloadHelper.MimeType
```
And use WKDownloadMimeType instead of MimeType in your viewDidLoad to see if it helps

**Comment by rogerwh** [comment]
It shows this error to me

![image](https://user-images.githubusercontent.com/56737987/184975129-e66ee5e6-6e34-4379-8285-2baa4ac6bb6b.png)

Do you know what can I do?
